viewing_party/party.py

```diff
@@ -1,23 +1,132 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating: 
+        return { "title": title, "genre": genre, "rating": rating}
+    else:
+        return None
+
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+
+def watch_movie(user_data, title):
+    watchlist = user_data["watchlist"]
+    watched = user_data["watched"]
+    for movie in watchlist:
+        if movie["title"] == title:
+            watchlist.remove(movie)
+            watched.append(movie)

```

Comment:
This would be a great place to use the `add_to_watched()` function you created earlier! That way you don't even need the `watched` variable in this function.